recipes/baumwelch/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "baumwelch" %}
+{% set version = "0.3.0" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.openfst.org/twiki/pub/Contrib/FstContrib/{{ name }}-{{ version }}.tar.gz
+  sha256: 01c2cc16c5eedcb28cdc2ae83659fc95bab0a0d32a7e9209fbecd59b8747d0f7
+  
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - openfst ==1.7.5
```

Comment:
Does this really need to be pinned so tightly?

Reply:
Historically, at least, yes. The versioning isn't "semantic"---each release breaks API compatibility.

Reply:
Oof! :frowning: